apps/src/templates/rubrics/ProgressRing.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import style from './rubrics.module.scss';
+import {learningGoalShape} from './rubricShapes';
+
+export default function ProgressRing({
+  learningGoals,
+  currentLearningGoal,
+  understandingLevels,
+  radius,
+  stroke,
+}) {
+  const normalizedRadius = radius - stroke * 2;
+  const circumference = normalizedRadius * 2 * Math.PI;
+  const assessedLearningGoals = understandingLevels.filter(u => u >= 0).length;
+  const currentLearningGoalOffset =
+    circumference -
+    ((currentLearningGoal + 1) / learningGoals.length) * circumference;
+  const assessedLearningGoalOffset =
+    circumference -
+    (assessedLearningGoals / learningGoals.length) * circumference;
+
+  return (
+    <svg height={radius * 2} width={radius * 2}>
+      <circle
+        className={style.progressRing}
+        stroke="#D4D5D7"
+        fill="transparent"
+        strokeWidth={stroke}
+        r={normalizedRadius}
+        cx={radius}
+        cy={radius}
+      />
+      <circle
+        className={style.progressRing}
+        stroke="#A9ACAF"
+        fill="transparent"
+        strokeWidth={stroke}
+        strokeDasharray={circumference + ' '}
+        style={{strokeDashoffset: currentLearningGoalOffset}}
+        r={normalizedRadius}
+        cx={radius}
+        cy={radius}
+      />
+      <circle
+        className={style.progressRing}
+        stroke="#0093A4"
```

Comment:
Consider using `color.light_primary_50` instead?

Reply:
Thanks - I need to familiarize myself with our colors! I'll get all of these changed.